src/components/Main/Results/ResultsCard.tsx

```diff
@@ -62,9 +62,32 @@ function ResultsCardFunction({ canRun, autorunSimulation, toggleAutorun, severit
     setUserResult(newUserResult)
   }
 
-  const hasResult = Boolean(result?.deterministicTrajectory)
-  const canExport = Boolean(hasResult)
+  const [hasResult, setHasResult] = useState<boolean>(false)
+  const [canExport, setCanExport] = useState<boolean>(false)
+  const scrollTargetRef = createRef<HTMLSpanElement>()
+
+  useEffect( () => {
+    if (result && result.deterministicTrajectory) {
+      setHasResult(true);
+      setCanExport(true);
+
+      const scrollTarget = scrollTargetRef.current
+      if (scrollTarget) {
+        scrollTarget.scrollIntoView({
+          behavior: 'smooth',
+          block: 'start',
+          inline: 'nearest'
+        })
+      }
+    } else {
+      setHasResult(false);
+      setCanExport(false);
+    }
+  }, [result, scrollTargetRef.current])
+
```

Comment:
This implementation provides a central authority for scroll behavior. The hook watches for changes on the Result object. Button states and scrolling behavior are computed accordingly.